controllers/seedimage_controller.go

```diff
@@ -326,16 +335,24 @@ func (r *SeedImageReconciler) reconcileConfigMapObject(ctx context.Context, seed
 	// (re)create the configmap
 	logger.V(5).Info("Create ConfigMap", "seedimage", seedImg.Namespace+"/"+seedImg.Name)
 
-	data := map[string][]byte{
+	binaryData := map[string][]byte{
 		"registration": regData,
 		"cloud-config": cloudConfig,
 	}
+
+	data := map[string]string{
+		ConfigMapKeyRegistrationURL: mRegistration.Status.RegistrationURL,
+		ConfigMapKeyDevice:          mRegistration.Spec.Config.Elemental.Install.Device,
+		ConfigMapKeyBaseImage:       seedImg.Spec.BaseImage,
+		ConfigMapKeyOutputName:      isoName,
```

Comment:
@frelon Correct me if I am wrong, the work to allow disk builds is based on this PR, hence this value will be revisited again around the disk feature right?

I am not happy having this isoName here, but I understand this is just a small refactor to allow custom builders and ease the development further improvements in that area.

Reply:
Yep, technically this PR is for the bring-your-own-OS so I didn't do any refactoring for ISO vs disk, but that is definitely on my list for the next PR!